index.js

```diff
@@ -9,38 +13,42 @@ class DuplicateCompatError extends Error {
   }
 }
 
-function load() {
-  // Recursively load one or more directories passed as arguments.
+/**
+ * Recursively load one or more directories passed as arguments.
+ *
+ * @param {string[]} dirs The directories to load
+ * @returns {object} All of the browser compatibility data
+ */
+function load(...dirs) {
   let dir,
     result = {};
 
-  function processFilename(fn) {
-    const fp = path.join(dir, fn);
-    let extra;
+  for (dir of dirs) {
+    dir = path.resolve(__dirname, dir);
```

Comment:
Hmm, `path.resolve()` will now be called many times, at each level of recursion. That seems harmless and things are still working, but it does make the code seem a bit confused. WDYT, can we do this once only?

Reply:
I think I could work up an improvement, yeah -- one moment!

Reply:
Think this no longer holds with the migration to `fdir()`.